README.md

```diff
@@ -98,8 +98,6 @@ participants:
     #  If the client combines the Beacon & validator nodes (e.g. Teku, Nimbus), then this list will also be passed to the combined Beacon-validator node
     validatorExtraParams: []
 
```

Comment:
You'll also want to add `builderNetworkParams: null` property here (to show users that it can exist),  add a comment explaining what it is, and then I'd add a sample of what the data structure looks like (since the default will be `nil`)